src/Screens/Onboarding/MentorList.tsx

```diff
@@ -31,36 +33,59 @@ interface OwnProps {}
 interface Props extends StateProps, DispatchProps, OwnProps {}
 
 const MentorList = (props: Props) => {
-  const [width, onLayout] = useWidth();
+  const [{ width, height }, onLayout] = useLayout();
   const measuredWidth = width || RN.Dimensions.get('window').width;
 
   return (
-    <Background onLayout={onLayout}>
-      <SafeAreaView style={styles.container}>
-        <RN.Text style={styles.title1}>Meet our</RN.Text>
-        <RN.Text style={styles.title2}>Mentors</RN.Text>
-        <RemoteData data={props.mentors} fetchData={props.fetchMentors}>
-          {value => (
-            <RN.View style={styles.carouselContainer}>
-              <snapCarousel.default
-                data={[...value.values()]}
-                renderItem={renderMentorCard}
-                sliderWidth={measuredWidth}
-                itemWidth={measuredWidth * 0.9}
-                inactiveSlideOpacity={1}
-                inactiveSlideScale={1}
-              />
-            </RN.View>
-          )}
-        </RemoteData>
-        <RN.View style={styles.bottom} />
+    <Background>
+      <SafeAreaView
+        style={styles.container}
+        forceInset={{ top: 'always', bottom: 'always' }}
+      >
+        <Message style={styles.title1} id="onboarding.mentorlist.upperTitle" />
+        <Message style={styles.title2} id="onboarding.mentorlist.lowerTitle" />
+        <RN.View onLayout={onLayout} style={styles.mentorListContainer}>
+          <RemoteData data={props.mentors} fetchData={props.fetchMentors}>
+            {value => (
+              <RN.View style={styles.carouselContainer}>
+                <snapCarousel.default
+                  data={[...value.values()]}
+                  renderItem={renderMentorCard(height)}
+                  sliderWidth={measuredWidth}
+                  itemWidth={measuredWidth * 0.9}
+                  inactiveSlideOpacity={1}
+                  inactiveSlideScale={1}
+                />
+              </RN.View>
+            )}
+          </RemoteData>
+        </RN.View>
+        <RN.View style={styles.bottom}>
+          <Button style={styles.button} onPress={() => {}}>
+            <Message
+              style={styles.buttonMessage}
+              id="onboarding.mentorlist.start"
+            />
+            <RN.Image source={require('../images/arrow.svg')} />
+          </Button>
```

Comment:
I am not sure about this button api. Now the button takes only onPress prop and children. My reasoning was that on the design there are different icons and texts with the button. I thought that this kinda api is more flexible than an api that takes more props and less children. Dunno.